com.unity.visualeffectgraph/Documentation~/VisualEffectGraphWindow.md

```diff
@@ -16,34 +16,35 @@ To open the Visual Effect Graph window, you can use any of the following methods
+| **Version Control** <br/> ![](Images/version-control.png)             | **Action**: When you enable [Version Control](https://docs.unity3d.com/Manual/Versioncontrolintegration.html), you can check out the changes you made in the asset through the toolbar.<br/><br/>**Dropdown**:<br/><br/>&#8226; **Get Latest**: Updates the asset on your machine to match changes you make in version control.<br/>&#8226; **Submit**: Submits the current state of the asset to the Version Control System.<br/>&#8226; **Revert**: Discards the changes you make to the asset.|
```

Comment:
I would say it differently:
When Version Control is enabled for a project, these buttons becomes available. The main action checks out the asset file.
* Get Latest: Update the asset file with latest changes from the repository
* Summit: ok
* Revert: Discards the changes you **made** to the asset.